class.jetpack.php

```diff
@@ -476,6 +483,11 @@ private function __construct() {
 			Jetpack_Heartbeat::init();
 		}
 
+		if ( Jetpack::is_active() ) {
```

Comment:
Perhaps we should do 
`if ( Jetpack::is_active() && defined( 'REST_REQUEST' ) ) {`

to limit these filters to rest request.